lib/gui/app.js

```diff
@@ -200,9 +200,21 @@ app.run(($timeout) => {
     // otherwise we risk presenting the same error over
     // and over again to the user, while also heavily
     // spamming our error reporting service.
+    // TODO: Continue scanning once drivelist is fixed (see below)
     driveScanner.stop();
 
-    return exceptionReporter.report(error);
+    // Clear the drive list, to avoid displaying stale data
+    $timeout(() => {
+      availableDrives.setDrives([]);
+    });
+
+    // TODO: Once drivelist can recover from transient failures
```

Comment:
This should be fixed by https://github.com/resin-io-modules/drivelist/pull/202 and can be removed once that's in & updated in Etcher